db/blip_handler.go

```diff
@@ -1350,7 +1353,7 @@ func (bh *blipHandler) processRev(rq *blip.Message, stats *processRevStats) (err
 	// bh.conflictResolver != nil represents an active SGR2 and BLIPClientTypeSGR2 represents a passive SGR2
 	forceAllowConflictingTombstone := newDoc.Deleted && (!bh.conflictResolver.IsEmpty() || bh.clientType == BLIPClientTypeSGR2)
```

Comment:
I think we actually may always allow conflicting tombstones now, or at least always in HLV mode or do we force this to be resolved on pull?